plugins/org.eclipse.reddeer.eclipse/src/org/eclipse/reddeer/eclipse/debug/ui/views/breakpoints/BreakpointsView.java

```diff
@@ -114,11 +132,10 @@ public void removeAllBreakpoints() {
 	public void importBreakpoints(String path) {
 		log.info("Importing breakpoints from '" + path + "'");
 		open();
-		new DefaultTree(cTabItem).setFocus();
-		new ContextMenuItem("Import Breakpoints...").select();
-		new DefaultShell("Import Breakpoints");
-		new LabeledText("From file:").setText(path);
-		new PushButton("Finish").click();
-		new WaitWhile(new ShellIsAvailable("Import Breakpoints"));
+		new DefaultTree(this).getContextMenu().getItem("Import Breakpoints...").select();
+		WizardDialog importBreakpointsDialog = new WizardDialog("Import Breakpoints");
+		importBreakpointsDialog.isOpen();
+		new LabeledText(importBreakpointsDialog, "From file:").setText(path);
+		importBreakpointsDialog.finish();
```

Comment:
Maybe add a check whether finish button is enabled? And if not, throw an exception containing dialog.getMessage() text? One can easily find out what went wrong (file does not exists, was not selected, etc... )

Reply:
We will need to create a specific implementation for that dialog because it shows the error message in an unusual place (not at the top, but at the bottom right next to finish/cancel buttons). Lets work in this in another issue.